plugins/org.eclipse.n4js/src/org/eclipse/n4js/internal/N4JSModel.java

```diff
@@ -125,6 +101,80 @@ public N4JSProject findProjectWith(URI nestedLocation) {
 		return null;
 	}
 
+	public URI convertToCorrespondingLocation(URI uri) {
```

Comment:
I would like to postpone that and do it during the follow-up (regarding better marker support for issues in external libraries). Reason is that this code is currently dead code here and is about to be changed during the follow-up. Also, this might be hlc/non-hlc sensitive.

Reply:
ok